scripts/libs.versions.toml

```diff
@@ -27,7 +27,6 @@ coil = "2.2.0"
 # @keep
 compileSdk = "33"
 compose-compiler = "1.4.0"
-compose-snapshot = "-"
```

Comment:
I believe this is currently unused and can be removed, but let me know if it serves a purpose.